src/roswire/common/msg.py

```diff
@@ -55,10 +50,42 @@ class Constant:
         The value of this constant.
     """
 
+    R_STRING_CONSTANT = re.compile("^\s*string\s+(\w+)\s*=\s*(.+)\s*$")
+    R_OTHER_CONSTANT = re.compile("^\s*(\w+)\s+(\w+)\s*=\s*([^\s]+).*$")
+
     typ: str
     name: str
     value: Union[str, int, float]
 
+    @classmethod
+    def from_string(cls, package: str, line: str) -> "Optional[Constant]":
+        """
+
+        Parameters
+        ----------
+        package: str
+            The name of the package that provides the constant.
+        line: str
```

Comment:
Bit of a nitpicky comment. Some of these lines end with a period whereas others don't. I have no particular preference, but we should probably be consistent.

Reply:
Done.